src/components/ui/popovers/Popovers.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
This seems to be covered by Notifications presentation page. Remove?

Reply:
I have removed popovers from notifications page